cli/root.go

```diff
@@ -96,20 +191,63 @@ func init() {
 	})
 }
 
+func initEndureConfig() *EndureConfig {
+	c := &config.Viper{
+		Path:   CfgFile,
+		Prefix: "rr",
+	}
+	err := c.Init()
+	if err != nil {
+		panic(err)
+	}
+
+	// init default config
+	if !c.Has(EndureKey) {
+		return &EndureConfig{
+			GracePeriod: time.Second * 30,
+			PrintGraph:  false,
+			RetryOnFail: false,
```

Comment:
I'm really eager to try this feature one day, but most of our services can retry internally :)

Reply:
`RetryOnFail` at the moment hidden from the user. There is no such key in the config. This is preparation to support endure `RetryOnFail` options but needs to be tested for races.